ios/klaviyo-react-native-sdk-configuration.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--
+   config.plist
+   Pods
+
+   Created by Ajay Subramanya on 9/18/24.
+   Copyright (c) 2024 Klaviyo. All rights reserved.
+-->
+<plist version="1.0">
+<dict>
+    <key>react_native_sdk_name</key>
+    <string>react_native</string>
+    <key>react_native_sdk_version</key>
+    <string>9.8.8</string>
```

Comment:
9.8.8 ?

Reply:
I think we have a release doc pertaining to RN, lets make sure we update it to mention this is another spot the version needs to get incremented before release

Reply:
🤦🏾  I'm glad I used a crazy version for testing. Fixed it to 1.0.0 assuming it's going to be the next release. Good call, I update the confluence doc with this step.